server/apps/api/models.py

```diff
@@ -10,6 +10,22 @@ class Domain(models.Model):
         null=False,
         blank=False,
     )
+
+    class Meta:
+        verbose_name = 'Domain'
+        verbose_name_plural = 'Domains'
+
+    def __str__(self):
+        return self.domain
+
+
+class Case(models.Model):
+    domain = models.ForeignKey(
+        Domain,
+        on_delete=models.CASCADE,
+        null=False,
+        blank=False,
```

Comment:
It's better to explicitly define `related_name='cases'` in order to be able to access to the cases of domain, like this:

```
domain = Domain.object.get(pk=1)
domain.cases.all()
```